packages/presentation/lib/src/pages/login_page/widgets/facebook_login_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/src/pages/login_page/bloc/login_bloc.dart';
+import 'package:presentation/style/colors.dart';
+import 'package:presentation/style/dimens.dart';
+import 'package:presentation/utils/images_container.dart';
+
+class FacebookLoginButton extends StatelessWidget {
```

Comment:
Why do you put a bloc in a public widget?

Reply:
can u explain?  How can I trigger any action in bloc without bloc?

Reply:
or u prefer to pass in to widget just a callback?